src/MICore/Transports/LocalLinuxTransport.cs

```diff
@@ -213,6 +197,42 @@ private void FifoWatcher_Deleted(object sender, FileSystemEventArgs e)
             }
         }
 
+        private bool GetRequiresRootAttach()
+        {
+            if (_miMode != MIMode.Clrdbg)
```

Comment:
Thanks. I missed that one. I thought you were talking about the one before we entered. I will take the one out on line 137.